seacrowd/sea_datasets/cc_aligned_doc/cc_aligned_doc.py

```diff
@@ -0,0 +1,152 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+from seacrowd.utils import schemas
+import datasets
+
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+_CITATION = """\
+@inproceedings{elkishky_ccaligned_2020,
+    author = {El-Kishky, Ahmed and Chaudhary, Vishrav and Guzm{\'a}n, Francisco and Koehn, Philipp},
+    booktitle = {Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing (EMNLP 2020)},
+    month = {November},
+    title = {{CCAligned}: A Massive Collection of Cross-lingual Web-Document Pairs},
+    year = {2020}
+    address = "Online",
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/2020.emnlp-main.480",
+    doi = "10.18653/v1/2020.emnlp-main.480",
+    pages = "5960--5969"
+}
+"""
+
+_DATASETNAME = "cc_aligned_doc"
+
+_DESCRIPTION = """\
+CCAligned consists of parallel or comparable web-document pairs in 137 languages aligned with English\
+(10 languages are from Southeast Asia; Burmese has two document collection with different scripts).\
+These web-document pairs were constructed by performing language identification on raw web-documents, \
+and ensuring corresponding language codes were corresponding in the URLs of web documents. This pattern \
+matching approach yielded more than 100 million aligned documents paired with English.
+"""
+
+_HOMEPAGE = "https://www2.statmt.org/cc-aligned/"
+
+_LANGUAGES = ["ind", "sun", "tha", "vie", "zlm", "lao", "khm", "mya", "ceb", "war"]
+
+_LICENSE = Licenses.UNKNOWN.value
+
+_LOCAL = False
+_SUBSETS = {"id_ID": "ind", "su_ID": "sun", "th_TH": "tha", "vi_VN": "vie", "ms_MY": "zlm", "lo_LA": "lao", "km_KH": "khm", "my_MM": "mya", "my_MM_zaw": "mya", "cx_PH": "ceb", "wy_PH": "war"}
+_URLS = {_DATASETNAME: "https://data.statmt.org/cc-aligned/en_XX-{subset}.tsv.xz"}
+
+_SUPPORTED_TASKS = [Tasks.MACHINE_TRANSLATION]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class CCAlignedDoc(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class CCAlignedDocDataset(datasets.GeneratorBasedBuilder):
```

Reply:
@ljvmiranda921 Thank you for the feedback, done the requested changes. 

Question regarding Dataset suffix: Is it needed for every dataset loader? Because in the template it is written: 

# TODO: Name the dataset class to match the **script name** using CamelCase instead of snake_case

So I thought that the class name should be CCAlignedDoc because the script name is cc_aligned_doc. If Dataset suffix should be for every data loader class I think we should change the instruction above in the template and make it clear. And also we should go through the merged data loaders because some of them are without Dataset suffixes (for example barasa, bible_en_id, casa, cc100 .....).

Reply:
Good point, agree that we should update the instructions to reflect this change.